desktop-exporter/package.json

```diff
@@ -9,6 +9,10 @@
         "react": "^18.2.0",
         "react-dom": "^18.2.0",
         "react-router-dom": "^6.4.3",
-        "react-window": "^1.8.8"
+        "react-window": "^1.8.8",
+        "@emotion/react": "^11.10.5",
+        "@emotion/styled": "^11.10.5",
+        "@mui/icons-material": "^5.10.15",
+        "@mui/material": "^5.10.15"
```

Comment:
We're not using these, or at least not yet. Let's remove them as dependencies for now.